modules/programs/gh.nix

```diff
@@ -112,9 +122,13 @@ in {
     xdg.configFile."gh/config.yml".source =
       yamlFormat.generate "gh-config.yml" cfg.settings;
 
-    programs.git.extraConfig.credential."https://github.com".helper =
-      mkIf cfg.enableGitCredentialHelper
-      "${cfg.package}/bin/gh auth git-credential";
+    programs.git.extraConfig.credential = let
+      hosts = lib.optional cfg.enableGitCredentialHelper "https://github.com"
+        ++ cfg.extraGitCredentialHelperHosts;
```

Comment:
Not sure about those semantics. `enableGitCredentialHelper` should control whether `programs.git.extraConfig.credential` is set at all, and there should be an option listing all the hosts to enable the credential helper for, including the default `github.com`.

Reply:
That makes a lot of sense, any name suggestions for that option?

Reply:
Maybe `gitCredentialHelper.enable` and `gitCredentialHelper.hosts`, and `mkRenamedOptionModule [ ... "enableGitCredentialHelper" ] [ ... "gitCredentialHelper" "enable" ]`.